README.md

```diff
@@ -3,54 +3,41 @@ Datashader
 
 [![Build Status](https://travis-ci.org/bokeh/datashader.svg)](https://travis-ci.org/bokeh/datashader)
 
-The Datashader project is focused on building better ways to visualize
-very large datasets, using intelligent server-side downsampling,
-automatic computations performed on the data as it progresses through
-the visualization pipeline, and other related techniques.  The project
-is under active development, and all the code and documentation is
+
+Datashader is a graphics pipeline system for creating meaningful
+representations of large amounts of data. It breaks the creation of images into
+2 steps:
+
+1. Aggregation
+
+   Data is binned per-pixel by various user specified methods into aggregates.
+
+2. Transformation
+
+   These aggregates are then further processed to create an image.
+
```

Comment:
I would add another step first, i.e. Projection, because that's done inside Datashader too (in Glyph, at the moment).  That way the README can be consistent with our other materials about datashading.